src/main/java/com/google/firebase/auth/OidcProviderConfig.java

```diff
@@ -102,7 +108,7 @@ CreateRequest getThis() {
     }
 
     void assertValidProviderIdFormat(String providerId) {
-      checkArgument(providerId.startsWith("oidc."), "Invalid OIDC provider ID: " + providerId);
+      checkOidcProviderId(providerId);
```

Comment:
Seems we can remove this helper method now and just call `checkOidcProviderId()` in its place.

Reply:
Restructured to make this possible.